web/styles/popovers.css

```diff
@@ -547,6 +547,26 @@ ul.popover-group-menu-member-list {
         font-size: 1.5em;
     }
 
+    #user-profile-streams-tab {
+        /* Assign the height for the tab content minus the 20px margin of container */
+        height: calc(60vh - 20px);
+        display: grid;
+        grid-template-rows: auto minmax(0, 1fr) auto;
```

Comment:
I guess would be better to use named grid areas, like we do in some other places.